mozilla-release/browser/app/profile/firefox.js

```diff
@@ -371,7 +371,10 @@ pref("browser.urlbar.usepreloadedtopurls.expire_days", 14);
   pref("browser.urlbar.update1", true);
   // If true, we show actionable tips in the Urlbar when the user is searching
   // for those actions.
-  pref("browser.urlbar.update1.interventions", true);
+
+  // CLIQZ-SPECIAL: DB-2432, in Cliqz Browser this always has to be false
+  // since we use our own extension dropdown.
+  pref("browser.urlbar.update1.interventions", false);
   // If true, we show new users and those about to start an organic search a tip
   // encouraging them to use the Urlbar.
   pref("browser.urlbar.update1.searchTips", true);
```

Comment:
May be turn off this also?

Reply:
Ah, no needs. It disabled for both our builds, beta and release. Please create a ticket for next version (1.35) to check this